### PR TITLE
feat(admin): person search (connects #285)

### PR DESCRIPTION
**Pull request by kiurchv**

### CODE REVIEW

File: packages/admin/src/pages/Persons/Search.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+const PHONE_PATTERN = "^\\+380\\d{9}$";
+
+const Search = ({ uri }) => (
+  <Box p={6}>
+    <Heading as="h1" fontWeight="normal" mb={1}>
+      Пошук пацієнтів
+    </Heading>
+    <Tabs.Nav>
+      <Tabs.Link to="./by_person_data">За даними</Tabs.Link>
```
Comment (ephor) [inline packages/admin/src/pages/Persons/Search.js]: Why underscore? We always use hyphens for urls

File: packages/admin/src/pages/Persons/Search.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        <>
+          <Router>
+            <Redirect from="/" to={`${uri}/by_person_data`} />
+            <SearchByPersonDataForm
+              path="by_person_data"
+              initialValues={locationParams}
+              onSubmit={setLocationParams}
+            />
+            <SearchByDeclarationForm
+              path="by_decalration"
```
Comment (ephor) [inline packages/admin/src/pages/Persons/Search.js]: typo in `decalration`

File: packages/admin/src/pages/Persons/Search.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+          label="Пошук за номером декларації"
+          placeholder="Введіть номер декларації"
+        />
+      </Box>
+    </Flex>
+    <input type="submit" hidden />
+  </Form>
+);
+
+const AuthnMethodsList = ({ data }) => (
```
Comment (ephor) [inline packages/admin/src/pages/Persons/Search.js]: Typo - ` Authn`

File: packages/admin/src/pages/Persons/Search.js [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+                    taxId,
+                    authenticationMethods,
+                    ...person
+                  }) => ({
+                    ...person,
+                    fullName: getFullName(person),
+                    birthDate: formatDate(birthDate),
+                    taxId: taxId || "—",
+                    authenticationMethods: (
+                      <AuthnMethodsList data={authenticationMethods} />
```
Comment (ephor) [inline packages/admin/src/pages/Persons/Search.js]: Typo - `Authn`

File: packages/admin/src/pages/Persons/Search.js (hunk 3 above)
Comment (kiurchv) [reply packages/admin/src/pages/Persons/Search.js]: Its not a typo

File: packages/admin/src/pages/Persons/Search.js (hunk 4 above)
Comment (kiurchv) [reply packages/admin/src/pages/Persons/Search.js]: Neither there

**Pull request merged**